modules/end-user-guide/partials/assembly_building-images-for-ides-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,18 @@
+:parent-context-of-building-images-for-ides-based-on-the-intellij-platform: {context}
+
+[id="building-images-for-ides-based-on-the-intellij-platform_{context}"]
+= Building images for IDEs based on the IntelliJ Platform
+
+:context: building-images-for-ides-based-on-the-intellij-platform
+
+This section describes how to build images for link:https://plugins.jetbrains.com/docs/intellij/intellij-platform.html#ides-based-on-the-intellij-platform[IDEs based on the IntelliJ Platform] up to version 2020.3.
```

Comment:
:) Now you can see one more benefit of my suggestion for `IntelliJ Platform-based IDEs` - the current line reads "based on the **IntelliJ Platform up to version 2020.3**".

Reply:
```suggestion
This section describes how to build images for link:https://plugins.jetbrains.com/docs/intellij/intellij-platform.html#ides-based-on-the-intellij-platform[IntelliJ Platform-based IDEs] up to version 2020.3.
```

Reply:
I find `IntelliJ Platform-based IDE` less understandable than `IDEs based on the IntelliJ Platform` so I prefer to keep it.

Reply:
@themr0c As you wish, but as I mentioned above, the text now says **IntelliJ Platform up to version 2020.3**.

Reply:
@themr0c I believe it's not possible to mention _**IDE based on the IntelliJ Platform up to version 2020.3**_ for 2 reasons:
1. Only JetBrains IDEs have versions 2020.3. Non-JetBrains IDEs that based on the IntelliJ Platform most likely have other versioning.
2. **_version 2020.3_** is the version of the IDEs, not of the IntelliJ Platform.
3. My suggestion is now corrected for these reasons.